libs/data-service-generator/src/util/ast.ts

```diff
@@ -406,13 +406,10 @@ export function classDeclaration(
   if (!decorators.length) {
     return declaration;
   }
-  const code = [
-    ...decorators.map((decorator) => recast.print(decorator).code),
-    recast.print(declaration).code,
-  ].join("\n");
-  const ast = parse(code);
-  const [classDeclaration] = ast.program.body as [namedTypes.ClassDeclaration];
-  return classDeclaration;
+
+  //@ts-ignore
```

Comment:
Let's remove the ts-ignore here, please.

Reply:
You cant because the types lib is not up to date